snap/snapcraft.yaml

```diff
@@ -8,11 +8,11 @@ description: |
 
 grade: stable
 confinement: strict
-base: core18
+base: core20
 
 apps:
   daemon:
-    command: bitcoind
+    command: bin/bitcoind
```

Comment:
Why are these paths changing? The new binary location shouldn't make a difference for releases, and this would just end up breaking users.

Reply:
I did not see the warning or error in the logs, but I do see that the install path remains the same.